hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/TestInvoker.java

```diff
@@ -201,14 +215,18 @@ public void testSdkXmlParsingExceptionIsTranslatable() throws Throwable {
   public void testExtractConnectTimeoutException() throws Throwable {
     throw extractException("", "",
         new ExecutionException(
-            new AmazonClientException(LOCAL_CONNECTION_TIMEOUT_EX)));
+            SdkException.builder()
+                .cause(LOCAL_CONNECTION_TIMEOUT_EX)
+                .build()));
   }
 
   @Test(expected = SocketTimeoutException.class)
   public void testExtractSocketTimeoutException() throws Throwable {
     throw extractException("", "",
         new ExecutionException(
-            new AmazonClientException(SOCKET_TIMEOUT_EX)));
+            SdkException.builder()
+                .cause(SOCKET_TIMEOUT_EX)
+                .build()));
   }
```

Comment:
since we added another method for completion exception, we should also test that here